framework/src/main/java/org/checkerframework/framework/util/element/TypeVarUseApplier.java

```diff
@@ -170,7 +170,8 @@ public void extractAndApply() {
 
     private boolean isBaseComponent(
             final AnnotatedArrayType arrayType, final Attribute.TypeCompound anno) {
-        return ElementAnnotationUtil.getTypeAtLocation(arrayType, anno.getPosition().location)
+        return ElementAnnotationUtil.getTypeAtLocation(
+                        arrayType, anno.getPosition().location, anno, false)
```

Comment:
pass null for anno, and check that addAnnotation handles null input (should do nothing)